prisma/seed.ts

```diff
@@ -3,10 +3,17 @@ import bcrypt from "@node-rs/bcrypt";
 
 const prisma = new PrismaClient();
 
-async function seed() {
+async function upsertDefaultUser() {
+  const userSync = await prisma.user.findFirst({
```

Comment:
we should be able to do a `prisma.user.upsert` here and just pass an empty object for `update`